newIDE/app/public/external/yarn/yarn-main.js

```diff
@@ -25,6 +25,7 @@ window.addEventListener('yarnReady', e => {
   yarn = e;
   yarn.app.fs = fs;
   yarn.app.electron = electron;
+  yarn.data.restoreFromLocalStorage(false);
```

Comment:
What is the `false` for?

Reply:
A new feature in yarn is to automatically save its entire edited file to localStorage. That way if its tab gets closed or electron window closed - later on when you start yarn again - it will restore its entire state of the app.

This is very useful to have on a web app and on mobile -it makes it act like vscode. It's however not useful when you bundle it in gdevelop, so I am disabling it. If you create a new file via gdevelop, it will get created from the backup in localStorage - unless I disable this.